### DISCUSSION TITLE
Is it possible to integrate AX Hybrid PRO?

### DISCUSSION THREAD

**Discussion by PablexXXXX** [petrleocompel/hikaxpro_hacs#101] [Q&A] [answered]
I have a panel DS PHA64 B and can´t integrate using this repository

**Comment by petrleocompel** [comment, marked as answer]
This is not an issue but discussion question. 

AxC Pro hybrid is AX Pro. 
Works. It is tested. 
If something does not work. Provide logs and Info about your system.
